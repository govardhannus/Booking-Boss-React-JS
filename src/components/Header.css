.headerContainer{
    display: flex;
}
.allproducts{
    text-align: left;
    font-weight: bold;
    font-size: 25px;
}