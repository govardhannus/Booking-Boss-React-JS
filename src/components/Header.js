import React from 'react';
import styles from './Header.scss'

class Header extends React.Component {
    render() {
        return (
            <div >
                <div className="allproducts">All products</div>
            </div>

        )
    }
}

export default Header;