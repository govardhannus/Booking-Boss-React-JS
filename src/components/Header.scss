.allproducts{
  margin-top: 5px;
  font-family: 'Roboto', 'Noto', sans-serif;
  font-size: 20px;
  opacity: 1.0;
}
.grid{
  display: grid;
}
.totalproducts{
  margin-top: 3px;
  font-family: 'Roboto', 'Noto', sans-serif;
  font-size: 10px;
  opacity: 0.5;;
}
.dropdown{
  position: absolute;
  right: 16px;
  border: none;
}
.dropdown2{
  position: absolute;
  margin-top: -20px;
  margin-left: -100px;
  height: 20px;
  width: 100px;

}