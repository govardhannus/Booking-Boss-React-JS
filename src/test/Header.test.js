import React from 'react';
import ReactShallowRender from 'react-test-renderer/shallow';
import Header from '../components/Header'
import product from '../components/products'

test('show render Header correctly', () => {
   const renderer =new ReactShallowRender();
    renderer.render(<Header/>);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});