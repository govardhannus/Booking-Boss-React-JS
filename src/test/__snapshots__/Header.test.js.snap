// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show render Header correctly 1`] = `
<div>
  <div
    className="allproducts"
  >
    All products
  </div>
  <div
    className="grid"
  >
    <div
      className="totalproducts"
    >
       24 products 
    </div>
  </div>
</div>
`;
